src/FormField/Input.php

```diff
@@ -41,6 +41,17 @@ class Input extends Generic
     public $actionLeft = null;
 
     /**
+     * Method similar to View::js() however will adjust selector
+     * to target the "input" element.
+     *
+     * $field->jsInput(true)->val(123);
+     */
+    public function jsInput($when = null, $action = null)
```

Comment:
Why not add this in FormField/Generic?
Currently it will not work for FormField/Checkbox for example.